### ISSUE TITLE
Activate Renovate or Dependabot

### ISSUE THREAD

**Issue by rockdaboot** [open-telemetry/opentelemetry-ebpf-profiler#108]
I already tried to activate it :) But that likely needs some org admin or maintainer.
Read [getting started](https://github.com/renovatebot/renovate/blob/main/docs/usage/getting-started/installing-onboarding.md)

[![Mend Renovate](https://app.renovatebot.com/images/banner.svg)](https://renovatebot.com)

**Comment by florianl** [comment]
Is there a document/guidance/request by OpenTelemetry to use Renovate?
I do see, that some OpenTelemetry repositories use Renovate while others don't. With depdendabot and golangci-lint the functionality of Renovate is already covered.
I'm not in favor for one over the other, as I'm more interested to have certain functionality covered and having it covered only once should be fine. If all solutions should be used, what is the policy if one disagrees with the other?

**Comment by dmathieu** [comment]
We definitely don't need both renovate and dependabot.
Use of one or the other is up to the SIG.

**Comment by florianl** [comment]
Closing as resolved with https://github.com/open-telemetry/opentelemetry-ebpf-profiler/pull/461.
